engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -3216,19 +3215,17 @@ class RuntimeServerTest
       )
     )
     context.receiveN(4) should contain theSameElementsAs Seq(
-      Api.Response(Api.BackgroundJobsStartedNotification()),
       Api.Response(requestId, Api.PushContextResponse(contextId)),
       Api.Response(
         Api.ExecutionUpdate(
           contextId,
           Seq(
             Api.ExecutionResult.Diagnostic.error(
               "Type error: expected `that` to be Number, but got quux (Unresolved_Symbol).",
-              None,
-              None,
+              Some(mainFile),
+              Some(model.Range(model.Position(10, 8), model.Position(10, 17))),
               None,
               Vector(
-                Api.StackTraceElement("Small_Integer.+", None, None, None),
```

Comment:
By inlining nodes for builtins that don't require `VirtualFrame` certain `BuiltinRootNode` are no longer on the stack. For example `+` is gone - I think it is OK as that better shows the location where the `+` is used. Is that OK, @4e6?